tests-manual/repro_issues_42.R

```diff
@@ -0,0 +1,41 @@
+library(shiny)
```

Comment:
hmmm, according to this https://github.com/dmpe/shinydashboard/commit/4dce160dcbabd9112a3230e881ae92b1bdf87238 it should be fixed. Will need to look on it later but maybe something with grund could have been forgotten ?